File: solutions/leetcode_268.swift
class Solution {
    func missingNumber(_ nums: [Int]) -> Int {
        var sum = 0
        for i in 0...nums.count {
            sum += i
        }
        for num in nums {
            sum -= num
        }
        return sum
    }
}


let given_nums = [9, 6, 4, 2, 3, 5, 7, 0, 1]
let expected = 8

let solution = Solution()
let result = solution.missingNumber(given_nums)
if result != expected {
    print("\(result) not equals \(expected)")
}

File: solutions/leetcode_268.py
class Solution:
    def missingNumber(self, nums: list[int]) -> int:
        return sum(range(len(nums) + 1)) - sum(nums)


if __name__ == '__main__':
    given_nums = [9, 6, 4, 2, 3, 5, 7, 0, 1]
    expected = 8

    solution = Solution()
    result = solution.missingNumber(given_nums)
    assert result == expected
